meta/main.yml

```diff
@@ -2,7 +2,7 @@
 galaxy_info:
   company: Idealista
   min_ansible_version: 2.2
```

Comment:
I think the minimum version is 2.4.x.x now

Reply:
done